CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ Exercism would be impossible without people like you being willing to spend time
 
 You'll need Go version 1.7 or higher. Follow the directions on http://golang.org/doc/install
```

Comment:
I know this was not part of your PR, but would you mind bumping this up to 1.8 or higher? Go v1.7 is no longer supported by dep so this document is not correct.